_15thnight/api/account.py

```diff
@@ -75,7 +75,7 @@ def forgot_password():
             user.reset_created_at < datetime.utcnow() - reset_token_life):
             user.generate_reset_token()
         user.save()
-        link = '%sreset-password/%s/%s' % (url_for('index', _external=True),
+        link = '%sreset-password/%s/%s' % (url_for('index', _external=True, _scheme="https"),
```

Comment:
Let's not hardcode the _scheme here. Let's use the `PREFERRED_URL_SCHEME` from the config here.